### PR TITLE
BPMSPL-690: Initial standalone image for Standalone Controller

### PR DESCRIPTION
**Pull request by rbarriuso**
Also updated product envs of business central and kie-server

### CODE REVIEW

File: standalonecontroller/image.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    - "/opt/eap/bin/standalone.sh"
+    - "-b"
+    - "0.0.0.0"
+    - "-c"
+    - "standalone-full.xml"
+scripts:
+    - package: standalonecontroller
+      exec: install
+sources:
+    - artifact: jboss-bpmsuite-7.0.0.test20170720-standalone-controller-ee7.zip
```
Comment (rbarriuso) [inline standalonecontroller/image.yaml]: This is a temporary artifact until the standalone controller product artifacts are available

File: standalonecontroller/image.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-standalonecontroller"
+description: "Red Hat JBoss BPM Suite 7.0 Standalone Controller container image"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite-standalonecontroller"
+    - name: "JBOSS_BPMSUITE_VERSION"
```
Comment (goldmann) [inline standalonecontroller/image.yaml]: Wasn't the conclusion to change this env variable too to follow what's in `JBOSS_PRODUCT`?

Comment (goldmann) [reply standalonecontroller/image.yaml]: In other words - shouldn't it be: `JBOSS_BPMSUITE_STANDALONECONTROLLER_VERSION`?

Comment (rbarriuso) [reply standalonecontroller/image.yaml]: Ops, I'll fix it

Comment (errantepiphany) [reply standalonecontroller/image.yaml]: Thanks @rbarriuso . Please let us know when it's ready so this PR can get processed. :)

File: standalonecontroller/image.yaml (hunk 1 above)
Comment (errantepiphany) [reply standalonecontroller/image.yaml]: @rbarriuso I see this has been uploaded into XXX . Thanks!

Comment (rbarriuso) [reply standalonecontroller/image.yaml]: Actually now I'm changing it to use the latest "official" artifact "jboss-brms-bpmsuite-7.0.0.ER-redhat-1-execution-server-controller-ee7.zip". I'll push it soon

**Comment by rbarriuso** [comment]
@goldmann I've also changed the images versions to match the artifacts used

File: standalonecontroller/image.yaml (hunk 1 above)
Comment (goldmann) [reply standalonecontroller/image.yaml]: Updated comment, don't leak internal resources.

File: businesscentral/image.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
 description: "Red Hat JBoss BPM Suite 7.0 Business Central container image"
-version: "7.0.0"
+version: "7.0.0.CAP02"
```
Comment (goldmann) [inline businesscentral/image.yaml]: This should be `7.0.0`.

File: businesscentral/image.yaml [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 from: "jboss-eap-7/eap70:latest"
 user: 185
 envs:
     - name: "JBOSS_PRODUCT"
-      value: "bpmsuite"
-    - name: "JBOSS_BPMSUITE_VERSION"
-      value: "7.0.0"
+      value: "bpmsuite-businesscentral"
+    - name: "JBOSS_BPMSUITE_BUSINESSCENTRAL_VERSION"
+      value: "7.0.0.CAP02"
```
Comment (goldmann) [inline businesscentral/image.yaml]: This should be `7.0.0`. Besides this you need to update all envs, not just one of them, if you do so.

**Review by goldmann — CHANGES_REQUESTED** [review]

File: businesscentral/image.yaml (hunk 4 above)
Comment (rbarriuso) [reply businesscentral/image.yaml]: Changing this env I just intended to express that the version of the inner artifact used was that one. The complete product version could still be a different one. In any case, I'll change all back to 7.0.0

**Comment by rbarriuso** [comment]
Changed versions

**Review by goldmann — APPROVED** [review]

**Pull request merged**